### ISSUE TITLE
[BUG] Application crashes when I open the Upload list

### ISSUE THREAD

**Issue by MegaBrutal** [owncloud/android#3044]
### Actual behaviour
Whenever I try to open the Upload list, the application immediately crashes, the Upload list never actually appears.

### Expected behaviour
The Upload list should open where I should be able to review the status of my uploads and retry uploads if they failed for any reason.
 
### Steps to reproduce
1. Open the application, or check for an upload notification on the Android notification list.
2. Navigate to the Upload list menu (second icon at the botton with the up arrow), or tap the Android upload notification.
3. See how the application crashes.

I really think it's something unique that only happens to me, I'm not yet figured out what special conditions I might have.

Can this problem be reproduced with the official owncloud server?
I'm not comfortable trying this right now.

### Environment data
Android version: 6.0.1

Device model: Samsung Galaxy S5 Neo SM-G903F

Stock or customized system: Stock (not rooted – yet)

ownCloud app version: 2.15.3 release 13511f58e

ownCloud server version: ownCloud 10.5.0 (stable)

#### Web server error log
No HTTP queries are made to the server when the application crashes.

#### ownCloud log (data/owncloud.log)
No new lines appear in owncloud.log.

I have a screenshot of the stack trace. I choose to send a crash report many times, but I don't know what happens to them and how could I possibly pair them to this report. Usually I mention "upload list" in the description.

![Screenshot_20201227-190508](https://user-images.githubusercontent.com/22104628/103177892-92bef780-487e-11eb-9f26-6331246bb760.png)

**Comment by jesmrec** [comment]
I've just checked it with an Android 6.0.1 device (not a Samsung) and i was not able to reproduce.

Probably @theScrabi or @abelgardep will find something in the logs you pasted. You can also send us complete logs by enabling the logging feature in the app. Go to the `Settings` view and tap 5 times in the version number on the bottom. Then, you'll the the `Logs` section just there.

Thanks!

**Comment by MegaBrutal** [comment]
Here is the log. Strangely the exception itself doesn't get logged here (Verbose level).
x-special/nautilus-clipboard
copy
file:///home/megabrutal/ownCloud/Documents/ghbug3044-ownCloud.log

I don't think it is generally reproducible, there must be some special circumstances on my phone, but I don't know how can it be reproduced, so I really hope the logs / stack traces help. Also I wonder what happens to the reports I send when Android offers me to send a crash report.

Anyway, the Upload list used to work. It stopped working when I made many photos and videos on an event in August. Then one time it started working again, I took the chance to restart my failed uploads at the time, but then something happened and it stopped working again.

**Comment by MegaBrutal** [comment]
[ghbug3044-ownCloud.log](https://github.com/owncloud/android/files/5805780/ghbug3044-ownCloud.log)

**Comment by jesmrec** [comment]
thanks for sharing logs! we'll take a look

**Comment by abelgardep** [comment]
Hi @MegaBrutal, we have released a new version: 2.16 

Could you check if this problem is still there? Thanks!

**Comment by MegaBrutal** [comment]
I upgraded to 2.16, but unfortunately it didn't solve the issue. The problem is the same as in the original description, there is no change.

This issue has been a nuisance for me during the past half a year, I regret not reporting it earlier. I thought someone else will notice and it will be fixed without me reporting it. I've been using ownCloud in the past 4 years, besides some occasional annoyance, generally it's pretty awesome. Therefore I posted a bounty for this issue. Whoever fixes it eventually should not forget to claim it. The project deserves the support.

https://www.bountysource.com/issues/95669852-bug-application-crashes-when-i-open-the-upload-list
